src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1157,10 +1157,11 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                                 toDispose.Add(DocumentIdWorker.GetLowerIdSliceAndStorageKey(context, attachment.Name, out _, out Slice attachmentName));
                                 toDispose.Add(DocumentIdWorker.GetLowerIdSliceAndStorageKey(context, attachment.ContentType, out _, out Slice contentType));
 
-                                var newChangeVector = ChangeVectorUtils.GetConflictStatus(attachment.ChangeVector, localAttachment?.ChangeVector) switch
+                                var localChangeVector = result.Attachment?.ChangeVector ?? result.Tombstone?.ChangeVector;
```

Comment:
let's move this to `AttachmentOrTombstone`

Reply:
done